components/Sidebar/index.tsx

```diff
@@ -76,10 +77,11 @@ const Sidebar = ({ isOpen }: IProps): JSX.Element => {
               )
             })
           }
+          <LzButton className='w-full sm:hidden' size='lg'><LzLink to='/'> Dashboard</LzLink></LzButton>
```

Comment:
By the way, you can actually make the button a link by passing the url via the `asLink` props.